src/aiocomelit/api.py

```diff
@@ -531,22 +361,22 @@ async def get_all_areas_and_zones(
         queries: dict[int, dict[str, Any]] = {
             1: {
                 "desc": "AREA description",
-                "page": "/user/area_desc.json",
+                "page": f"/user/{self._vedo}area_desc.json",
```

Comment:
It would be cleaner to maybe make this either a property you overwrite, rather than rely on string concatenation, as it allows for easier expansion

Reply:
In the end I judged that using a postfix in the url is a good enough approach to make code easy to maintain and readable at the same time.